app/src/main/java/org/mozilla/fenix/compose/list/ExpandableListHeader.kt

```diff
@@ -64,10 +61,10 @@ fun ExpandableListHeader(
                 .padding(horizontal = 16.dp, vertical = 8.dp),
             verticalAlignment = Alignment.CenterVertically,
         ) {
-            PrimaryText(
```

Comment:
Probably want to file something separate to remove PrimaryText and SecondaryText